mainnet-1/specs/stargaze.json

```diff
@@ -0,0 +1,787 @@
+                "rule": {
+                  "block": 14600
+                }
+              }
+            ]
+          }
+        ],
+        "contributor": [
+          "lava@1jnsszemt05kcxd4pf3uzewl9xqtzfpfg65960w"
+        ],
+        "contributor_percentage": "0.025",
+        "identity": "7C0AE2E5F08269E6"
+      },
+      {
+        "index": "STRGZT",
+        "name": "stargaze testnet",
+        "enabled": true,
+        "imports": [
+          "STRGZ"
+        ],
+        "reliability_threshold": 268435455,
+        "data_reliability_enabled": true,
+        "block_distance_for_finalized_data": 0,
+        "blocks_in_finalization_proof": 1,
+        "average_block_time": 5800,
+        "allowed_block_lag_for_qos_sync": 2,
+        "shares": 1,
+        "min_stake_provider": {
+          "denom": "ulava",
+          "amount": "15000000000"
+        },
+        "api_collections": [
+          {
+            "enabled": true,
+            "collection_data": {
+              "api_interface": "rest",
+              "internal_path": "",
+              "type": "GET",
+              "add_on": ""
+            },
+            "apis": [],
+            "headers": [],
+            "inheritance_apis": [],
+            "parse_directives": [],
+            "verifications": [
+              {
+                "name": "chain-id",
+                "values": [
+                  {
+                    "expected_value": "elgafar-1"
+                  }
+                ]
+              }
+            ]
+          },
+          {
+            "enabled": true,
+            "collection_data": {
+              "api_interface": "grpc",
+              "internal_path": "",
+              "type": "",
+              "add_on": ""
+            },
+            "apis": [],
+            "headers": [],
+            "inheritance_apis": [],
+            "parse_directives": [],
+            "verifications": [
+              {
+                "name": "chain-id",
+                "values": [
+                  {
+                    "expected_value": "elgafar-1"
+                  }
+                ]
+              }
+            ]
+          },
+          {
+            "enabled": true,
+            "collection_data": {
+              "api_interface": "tendermintrpc",
+              "internal_path": "",
+              "type": "",
+              "add_on": ""
+            },
+            "apis": [],
+            "headers": [],
+            "inheritance_apis": [],
+            "parse_directives": [],
+            "verifications": [
+              {
+                "name": "chain-id",
+                "values": [
+                  {
+                    "expected_value": "elgafar-1"
+                  }
+                ]
+              }
+            ]
+          }
+        ],
+        "contributor": [
+          "lava@1jnsszemt05kcxd4pf3uzewl9xqtzfpfg65960w"
+        ],
+        "contributor_percentage": "0.025",
+        "identity": "7C0AE2E5F08269E6"
```

Comment:
This should be the logo of Axelar, if you don't have it, you can leave empty (= removed)

Reply:
The field value has been updated [f945313](https://github.com/lavanet/lava-config/pull/24/commits/f945313bd76273364a1dd62f8aa8357432ed92d8)